ci/code_checks.sh

```diff
@@ -121,82 +121,82 @@ if [[ -z "$CHECK" || "$CHECK" == "patterns" ]]; then
     # Check for imports from pandas.core.common instead of `import pandas.core.common as com`
     # Check for imports from collections.abc instead of `from collections import abc`
     MSG='Check for non-standard imports' ; echo $MSG
-    invgrep -R --include="*.py*" -E "from pandas.core.common import" pandas
-    invgrep -R --include="*.py*" -E "from pandas.core import common" pandas
-    invgrep -R --include="*.py*" -E "from collections.abc import" pandas
-    invgrep -R --include="*.py*" -E "from numpy import nan" pandas
+    invgrep -R --include=*.py* -E "from pandas.core.common import" pandas
+    invgrep -R --include=*.py* -E "from pandas.core import common" pandas
```

Comment:
are you sure this works? this generally needs to be quoted

Reply:
I don't mind reverting.
I tested it on my machine.
```
Grub version = grep (GNU grep) 3.3

Bash version = GNU bash, version 5.0.11(1)-release (x86_64-pc-linux-gnu)
```

Plus, we have a few variations of the ```--include```/```--exclude```  already in the CI.
For example:

https://github.com/pandas-dev/pandas/blob/facd756fb0e6a2d9a8ac934ae4295c40a0c30102/ci/code_checks.sh#L203

https://github.com/pandas-dev/pandas/blob/facd756fb0e6a2d9a8ac934ae4295c40a0c30102/ci/code_checks.sh#L208

But if it's too risky, I'll just change everything to the ```safe``` syntax.

Reply:
this doesn’t work in macosx and likely not on windows (though not sure how much of this works on windows)

Reply:
Ok, so revert to quotes?
And if the answer is 'yes', double or single quotes?

Reply:
yes quotes; single quotes i think are appropriate here (but likely double do the same)

Reply:
@jreback, the problem now is that it doesn't work:

```invgrep -R --include='*.{py,pyx}' 'self' pandas```

or

```invgrep -R --include="*.{py,pyx}" 'self' pandas```

(not even if I switch the quotes on the regex and the wanted string, tried all four combinations)

should I just "exand" each statement?, meaning making every ```--include=*.{py,pyx}``` to ```--include='*.py' --include='*.pyx'```

Reply:
@MomIsBestFriend right I am not sure the expansion syntax is generally used on linux, ok with just making it a substion though, e.g.
```INCLUDES=--include='*.py'......``` and then using INCLUDES (and EXCLUDES) might be simpler